lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -479,6 +479,13 @@ def test_ccx_invite_enroll_up_to_limit(self):
         students = [
             UserFactory.create(is_staff=False) for _ in range(3)
         ]
+        admins = [
```

Comment:
Instead of creating `admins` here, can you change the above line to

    students = [self.instructor, self.staff] + [...